prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -117,6 +124,13 @@ static class FeaturePackOrDefinition {
                 order = 3
         )
         Optional<Path> definition;
+
+        @CommandLine.Option(
+                names = CliConstants.LIST_PROFILES,
```

Comment:
when introducing new parameter you need to update the help message. See `prospero install -h`. For the install command syntax is printed manually, it needs to be updated too.